assets/css/framework.css

```diff
@@ -0,0 +1,243 @@
+@import url(reset.css);
+
+
+.row {
+  display: grid;
+  grid-template-columns: repeat(12, 1fr); }
+
+.col-1 {
+  grid-column: span 1; }
+
+.col-2 {
+  grid-column: span 2; }
+
+.col-3 {
+  grid-column: span 3; }
+
+.col-4 {
+  grid-column: span 4; }
+
+.col-5 {
+  grid-column: span 5; }
+
+.col-6 {
+  grid-column: span 6; }
+
+.col-7 {
+  grid-column: span 7; }
+
+.col-8 {
+  grid-column: span 8; }
+
+.col-9 {
+  grid-column: span 9; }
+
+.col-10 {
+  grid-column: span 10; }
+
+.col-11 {
+  grid-column: span 11; }
+
+.col-12 {
+  grid-column: span 12; }
+
+body {
+  font-family: 'Roboto Condensed'; }
+
+.text-center {
+  text-align: center; }
+
+.text-left {
+  text-align: left; }
+
+.text-right {
+  text-align: right; }
+
+h1, h2, h3 {
+  font-size: 10px; }
+
+a {
+  text-decoration: cyan; }
```

Comment:
Why did you make text-decoration, cyan for all anchor tags, please remember this is a framework that should be a bit more abstract, think, am I going to have a tag cyan for most usages of this framework? Probably not.

Good for trying to make it more unique though. Also, since you are only adding color, `text-decoration-color` is fine.